src/main/java/frc/robot/RobotContainer.java

```diff
@@ -132,13 +134,8 @@ private void configureBindings() {
     controller.button(11).whileTrue(new StartEndCommand(arm::extend, arm::stop, arm));
     controller.button(12).whileTrue(new StartEndCommand(arm::retract, arm::stop, arm));
 
-    controller
-        .button(122)
-        .whileTrue(
-            new SequentialCommandGroup(
```

Comment:
Again, we should consider bringing back the prelevel command now that auto level works.

Reply:
We need it for auto. we could do the same for teleop, we could test it. Not really sure why it didn't work. Although I am not sure if sequential command works the way I think it works. We could just call prelevel command in auto level command directly fi we want to use it